docs/src/developer-guide/package-json-conventions.md

```diff
@@ -10,22 +10,16 @@ The following applies to the "scripts" section of `package.json` files.
 
 npm script names MUST contain only lower case letters, `:` to separate parts, `-` to separate words, and `+` to separate file extensions. Each part name SHOULD be either a full English word (e.g. `coverage` not `cov`) or a well-known initialism in all lowercase (e.g. `wasm`).
 
-Here is a summary of the proposal in EBNF.
-
-```ebnf
-name = life-cycle | main ":fix"? target? option* ":watch"?
-
-life-cycle = prepare | preinstall | install | postinstall | prepublish | preprepare | prepare | postprepare | prepack | postpack | prepublishOnly;
-
-main = "build" | "lint" | "start" | "test";
-
-target = ":" word ("-" word)* | extension ("+" extension)*;
-
-option = ":" word ("-" word)*;
-
-word = [a-z]+;
-
-extension = [a-z0-9]+;
+Here is a summary of the proposal in ABNF.
+
+```abnf
+name         = life-cycle / main target? option* ":watch"?
+life-cycle   = "prepare" / "preinstall" / "install" / "postinstall" / "prepublish" / "preprepare" / "prepare" / "postprepare" / "prepack" / "postpack" / "prepublishOnly"
+main         = "build" / "lint" ":fix"? / "release" / "start" / "test"
+target       = ":" word ("-" word)* / extension ("+" extension)*
+option       = ":" word ("-" word)*
+word         = ALPHA +
+extension    = ( ALPHA / DIGIT )+
 ```
```

Comment:
This was a rough sketch working from memory. I looked up the grammar for ABNF and rewrote it correctly.